src/openrct2/ride/Vehicle.cpp

```diff
@@ -1955,114 +1954,114 @@ static SoundIdVolume sub_6D7AC0(SoundId currentSoundId, uint8_t currentVolume, S
  *
  *  rct2: 0x006D77F2
  */
-static void vehicle_update(Vehicle* vehicle)
+void Vehicle::vehicle_update()
 {
     // The cable lift uses the ride type of NULL
-    if (vehicle->ride_subtype == RIDE_TYPE_NULL)
+    if (this->ride_subtype == RIDE_TYPE_NULL)
     {
-        cable_lift_update(vehicle);
+        cable_lift_update(this);
         return;
     }
 
-    auto rideEntry = get_ride_entry(vehicle->ride_subtype);
+    auto rideEntry = get_ride_entry(this->ride_subtype);
     if (rideEntry == nullptr)
         return;
 
-    auto ride = get_ride(vehicle->ride);
-    if (ride == nullptr)
+    auto r = get_ride(ride);
```

Comment:
`curRide` might be a better name but this is fine as well.

Reply:
right so I named it `r` because that is how it was implemented in other places, i will change the naming convention to `curRide` in the whole `vehcle.cpp` file.

Reply:
The variable names should be consistent but don't just blindly change them all. Context is key.